scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua

```diff
@@ -12,6 +14,16 @@ function onMobDeath(mob, player, isKiller)
 end;
 
 -----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setMod(MOD_DOUBLE_ATTACK, 20);
+	mob:setMod(MOD_STORETP, 10);
```

Comment:
2nd line is a tab..
Set your editor to replace tabkey with 4 spaces.

Don't use lousy github website to edit files, nor their lousy github program for windows, or do copy&paste from someplace like pastebin or windows notepad/wordpad, use something like notepad++ or visual studio's editor. Otherwise weird artifacts get inserted into formatting like this one stray tab.